app/main/views/suppliers.py

```diff
@@ -200,6 +200,9 @@ def supplier_search():
     q = q.order_by(*ob)
     results = list(q)
 
+    # remove 'hidden' example listing from result
+    results = [_ for _ in results if _.id > 0]
```

Comment:
The way it works in the metrics is filtering by ABN magic number https://github.com/AusDTO/dto-digitalmarketplace-api/blob/master/app/models.py#L502

Suggest doing the same here.